python/lsst/sconsUtils/scripts.py

```diff
@@ -643,6 +643,14 @@ def s(l):
             pyList = [control.runPythonTests(pyList)]
         else:
             pyList = []
+
+        predecessors = []
+        for single in pySingles:
+            for pyTest in pyList:
+                state.env.Depends(pyTest, single)
+            for node in predecessors:
```

Comment:
It may be too early in the morning for me but how does predecessors ever get filled? Line 647 initializes it but then I can't see anything that populates it.

Reply:
Oops, forgot to append the current `single` to it.

Reply:
And actually I only need one predecessor, rather than all.